main.py

```diff
@@ -0,0 +1,41 @@
+class Chainsaw:
+    """
+    Class exact to the one from java first lab
+    """
+    instance = None
+
+    def __init__(self, brand="T-800", power=1100, fuel_tank_capacity=3.7, fuel_level=3.3, is_working=False):
+        self.__brand = brand
+        self.__power = power
+        self.__fuel_tank_capacity = fuel_tank_capacity
+        self.__fuel_level = fuel_level
+        self.__is_working = is_working
+
+    def __str__(self):
```

Comment:
If you want to make multiline string, it's not mandatory to make line continuation characters (\\). You can just use tripple-quote string or simply continue string on new line without the character. Not a mistake though. Just a recommentdation